.travis.yml

```diff
@@ -69,6 +71,36 @@ matrix:
      script:
       - RUSTFLAGS="-D warnings" cargo build --release --verbose
       - RUSTFLAGS="-D warnings" cargo test --verbose
+   - language: android
+     dist: trusty
+     android:
+       components:
+        - build-tools-26.0.1
+        # Minimum API level supported
+        - android-21
+     install:
+      # Install rust manually
+      - curl https://build.travis-ci.org/files/rustup-init.sh -sSf | sh -s -- -y --default-toolchain stable
+      - export PATH=$HOME/.cargo/bin:$PATH
```

Comment:
Should probably just put this in an `env` section https://docs.travis-ci.com/user/environment-variables/

Reply:
We still need to install `rust` this way unless we have a way to have an environment with 2 language (android and rust).